blazor/pivot-table/grouping.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 Grouping is the most-useful feature in pivot table and the component automatically groups date, time, number and string. For example, the date type can be formatted and displayed based on year, quarter, month, and more. Likewise, the number type can be grouped range-wise, such as 1-5, 6-10, etc. These group fields will act as individual fields and allows users to drag them between different axes such as columns, rows, values, and filters and create pivot table at runtime.
 
+To know about **Grouping** in Blazor Pivot Table component, you can check on this video.
```

Comment:
To have a quick glance on how to group row and column field items in the Blazor Pivot Table component, watch this video:


Move this video section to line no:28.